Packages/vcs/Lib/Canvas.py

```diff
@@ -74,7 +74,7 @@
 import vcs.manageElements
 
 class SIGNAL(object):
-    
```

Comment:
FWIW, it's generally a bad idea to mix large whitespace-only (or other maintenance-oriented) changes with actual code changes.

It makes it more difficult for reviewers to find the actual changes that need reviewing, and leads to odd VCS histories (for instance, changing non-VTK related files in a topic that is focused on cleaning up VTK usage).